README.md

```diff
@@ -39,10 +48,58 @@ Or install it yourself as:
     $ gem install ar2dto
 
 ## Usage
-TBD
+
+In the following sections, we explain the API provided by the gem and some basic usage.
 
 ### Setting up your models
-TBD
+
+To use the gem, you need to add `has_dto` to your ActiveRecord models. For example:
+```ruby
+class User < ApplicationRecord
+  has_dto
+end
+```
+
+This will dynamically create a class called `UserDTO` and will add two methods to your ActiveRecord model: `#to_dto` and `.to_dto`.
+
+### DTO class
+
+This class is dynamically created based on your models that declare `has_dto`. The goal of these classes is to be data-only. They don't have access to the DB, business logic, or calculate things on the fly. They just store the data and provide them in a read-only fashion through plain simple methods.
```

Comment:
I think about them as `jsons` but I don't know if referring to json improves the understanding of what dtos are 🤔

Reply:
hmm I get your point, but I'm not sure that's necessarily true, I can have an XML represent a DTO I guess?